frontend/src/components/Layout/Menu/styled.ts

```diff
@@ -20,7 +20,7 @@ export const MenuTabs = styled('div')({
   borderRight: '1px solid rgba(0, 0, 0, 0.39)',
   boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.1)',
```

Comment:
i guess, we can remove 'px' here as well. As in other styled files

Reply:
yep